src/pages/proposal.tsx

```diff
@@ -371,9 +373,9 @@ export default function ProposalPage() {
               {proposal?.type === ProposalType.POI && <ProposalFooterPoi configuration={proposal.configuration} />}
               {showTenderProposals && <TenderProposals proposals={tenderProposals.data} />}
               {showBidProposals && <BidProposals proposals={bidProposals.data} />}
-              {proposal && isBiddingAndTenderingProposal(proposal.type) && (
-                <BiddingAndTenderingProcess proposal={proposal} tenderProposalsTotal={tenderProposals?.total} />
-              )}
+              {proposal && proposal.type === ProposalType.Pitch && <AboutPitchProcess proposal={proposal} />}
```

Comment:
How about something like

`proposal & <BiddingAndTenderingProcess proposal={proposal} />`

and 

```typescript
export default function BiddingAndTenderingProcess({ proposal }: Props) {
  switch (proposal.type) {
    case ProposalType.Pitch:
      return <AboutPitchProcess proposal={proposal} />
    case ProposalType.Tender:
      return <AboutTenderProcess proposal={proposal} />
    case ProposalType.Bid:
      return <AboutBidProcess proposal={proposal} />
    default:
      return null
  }
}
```

Just so proposal.tsx doesn't get too much bigger